ci-operator/jobs/openshift/openshift-azure/openshift-openshift-azure-presubmits.yaml

```diff
@@ -54,3 +54,58 @@ presubmits:
           - secret:
               name: cluster-secrets-azure
     trigger: ((?m)^/test( all| build),?(\s+|$))
+  - agent: kubernetes
+    always_run: false
+    branches:
+    - master
+    context: ci/prow/upgrade
+    decorate: true
+    labels:
+      artifacts: images
+    name: pull-ci-openshift-azure-upgrade
+    rerun_command: /test upgrade
+    skip_cloning: true
+    spec:
+      containers:
+      - args:
+        - --artifact-dir=$(ARTIFACTS)
+        - --give-pr-author-access-to-namespace=true
+        - --secret-dir=/usr/local/e2e-azure-secret
+        - --template=/usr/local/e2e-azure
+        - --target=upgrade
+        command:
+        - ci-operator
+        env:
+        - name: CONFIG_SPEC
+          valueFrom:
+            configMapKeyRef:
+              key: master.json
+              name: ci-operator-openshift-openshift-azure
+        - name: JOB_NAME_SAFE
+          value: upgrade
+        - name: CLUSTER_TYPE
+          value: azure
+        image: ci-operator:latest
+        name: test
+        resources:
+          limits:
+            cpu: 500m
+          requests:
+            cpu: 10m
+        volumeMounts:
+        - mountPath: /usr/local/e2e-azure
+          name: job-definition
+          subPath: cluster-launch-e2e-azure.yaml
```

Comment:
Is this right?

Reply:
No, https://github.com/openshift/release/pull/1394